static/src/js/zustand/types.ts

```diff
@@ -127,6 +127,25 @@ export type FacetParamsSlice = {
   }
 }
 
+export type FocusedCollectionSlice = {
+  /** The Focused Concept Slice of the store */
+  focusedCollection: {
+    /** The currently focused collection */
+    focusedCollection: string | null
+
+    /** Function to change the focused collection and fetch data */
+    changeFocusedCollection: (collectionId: string | null) => void
+    /** Function to get the focused collection metadata */
+    getFocusedCollection: () => void
+    /** Function to set the focused collection */
+    setFocusedCollection: (collectionId: string | null) => void
```

Comment:
Maybe worth commenting that calling the function with no args will unset. Not sure if that's totally obvious.